fmc/src/flow/pcr.rs

```diff
@@ -61,11 +69,63 @@ fn extend_pcr_common(env: &mut FmcEnv, hand_off: &HandOff, pcr_id: PcrId) -> Cal
     let rt_tci: [u8; 48] = okref(&rt_tci)?.into();
     env.pcr_bank.extend_pcr(pcr_id, &mut env.sha384, &rt_tci)?;
 
+    log_pcr(
+        &mut env.pcr_bank,
+        PcrLogEntryId::RtTci,
+        1 << pcr_id as u8,
+        &rt_tci,
+    )?;
+
     // Extend FW Image Manifest
     let manifest_digest = Tci::image_manifest_digest(env, hand_off);
     let manifest_digest: [u8; 48] = okref(&manifest_digest)?.into();
     env.pcr_bank
         .extend_pcr(pcr_id, &mut env.sha384, &manifest_digest)?;
 
+    log_pcr(
+        &mut env.pcr_bank,
+        PcrLogEntryId::FwImageManifest,
+        1 << pcr_id as u8,
+        &manifest_digest,
+    )
+}
+
+pub fn log_pcr(
+    pcr_bank: &mut PcrBank,
+    pcr_entry_id: PcrLogEntryId,
+    pcr_ids: u32,
```

Comment:
If this function won't be used elsewhere in FMC, could make it non-public and change pcr_id to just accept a single value.